### ISSUE TITLE
[Accessibility] - Screenreader does not announce Previous / Next month properly when Previous/Next action icons receive focus.

### ISSUE THREAD

**Issue by jennylgh** [Vuepic/vue-datepicker#132]
**Describe the bug**
When datepicker header previous/next icon (ActionIcon) receives focus, screenreader is not announcing the content properly.

**To Reproduce**
1. Go to DatePicker[ Documentation page for inline mode.](https://vue3datepicker.com/api/props/#inline)
2. Enable screenreader (Mac VoiceOver, or Windows NVDA). 
3. Tabbing into datepicker header to focus on Previous or Next button
4. Listen how screenreader announces the tag.

**Expected behavior**
1. For Previous button, it should read "Previous Month button" - this is necessary so user who depends on assistive technology can use keyboard (Spacebar or Enter) to activate the button.
2. For Next button, it should read "Next Month button" 

**Actual behavior**
1. For Previous button, it announces "Previous Month group".
2. For Next button, it only announces "group".

**Screenshots**
<img width="718" alt="image" src="https://user-images.githubusercontent.com/5761640/189202488-597780b0-78c1-4aad-9e09-ecfe14bb1a6a.png">

**Desktop & mobile (please complete the following information):**
 - Browser [e.g. chrome, safari] - Tested with Mac VoiceOver with Chrome and Safari. The issue is likely to occur for all screen readers (NVDA, JAWS) and browsers.
 - Library version [e.g. 3.0.0] - 3.7.4
